tinygrad/runtime/ops_torch.py

```diff
@@ -26,5 +26,5 @@ def __init__(self, size:int, dtype:DType, buf:Optional[torch.Tensor]=None): supe
   def fromCPU(cls, x):
     buf = torch.from_numpy(x).requires_grad_(False).to(device)
     return cls(prod(x.shape), type_map[buf.dtype], buf)
-  def toCPU(self): return self._buf.cpu().numpy()
+  def toCPU(self): return self._buf.numpy()
```

Comment:
I think cpu is still needed here in case the torch tensor is not on the cpu

Reply:
Thanks for the catch. Will it make sense to just use `force=True`? Which would do
`t.detach().cpu().resolve_conj().resolve_neg().numpy()`

Reply:
could just leave it as is, don't think any issues has been brought up that requires the force flag addition here

Reply:
This is the torch buffer, not the tinygrad buffer!